src/pages/groups/ListGroups.vue

```diff
@@ -7,8 +7,7 @@
             <div class="admin-page-header mr-4">Groups</div>
             <PermissionGuard :requiredRole="ROLES.SITE_ADMIN">
               <PvButton
-                v-tooltip.bottom="currentSite === 'any' ? 'Please select a specific site to add a group' : ''"
-                :disabled="currentSite === 'any'"
+                v-tooltip.bottom="currentSite === 'any' && 'Please select a specific site to add a group'"
```

Comment:
Why is the disabled prop removed? We're trading a one-liner for several lines in the submission function